.dev/hooks/pre-commit

```diff
@@ -19,7 +19,7 @@ include(joinpath(toplevel_directory, ".dev", "clima_formatter_options.jl"))
 needs_format = false
 
 for filename in readlines(`git diff --name-only --cached`)
-    endswith(filename, ".jl") || continue
+    (isfile(filename) && endswith(filename, ".jl")) || continue
```

Comment:
We may want to use `git diff --name-status --cached` to determine if a file is deleted or not.  Because a file can be removed from the index but still exist.

Reply:
Good call. Will update

Reply:
Thanks for fixing this!

Reply:
Update looks good!